packages/runtime/src/index.js

```
@@ -1,47 +1,64 @@
+              document.body.appendChild(elemento)
+            })
+            .catch(function (error) {
+              console.error(error)
+            })
+        } catch (error) {
+          console.error(error)
+        }
+      }
+    }
+  }
+  let etiquetasH = document.getElementsByTagName('div')
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'etiquetasH' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)